modules/datastore/tests/src/EventSubscriber/SubscriberTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Drupal\Tests\datastore\EventSubscriber;
+
+use Drupal\common\Resource;
+use Drupal\Core\Logger\LoggerChannelFactory;
+use Drupal\datastore\EventSubscriber\Subscriber;
+use Drupal\datastore\Service;
+use Drupal\metastore\Events\Registration;
+use Drupal\Tests\common\Traits\ServiceCheckTrait;
+use MockChain\Chain;
+use MockChain\Options;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\DependencyInjection\Container;
+
+/**
+ *
+ */
+class SubscriberTest extends TestCase {
+  use ServiceCheckTrait;
+
+  /**
+   *
+   */
+  public function test() {
+    $url = 'http://hello.world/file.csv';
+    $resource = new Resource('http://hello.world/file.csv', 'text/csv');
```

Comment:
Probably meant to re-use `$url`:

`$resource = new Resource($url, 'text/csv');`